web/tests/choice-params-with-default.spec.ts

```diff
@@ -31,3 +31,33 @@ test('test pagerank default choice values', async () => {
   expect(defaultHistogram).not.toEqual(incomingHistogram);
   state.close();
 });
+
+test('editing pagerank default choice values', async () => {
+  await workspace.editBox('pr1', { direction: 'all edges' }); // change direction
+  await workspace.editBox('pr2', { direction: 'all edges' }); // change direction
+  let boxEditor = await workspace.openBoxEditor('pr1');
+  await expect(boxEditor.getParameter('direction', 'select')).toHaveValue('string:all edges');
+  await boxEditor.close();
+  boxEditor = await workspace.openBoxEditor('pr2');
+  await expect(boxEditor.getParameter('direction', 'select')).toHaveValue('string:all edges');
+  await boxEditor.close();
+});
+
+test('multi-choice default values', async () => {
+  await workspace.addBox({ id: 'ex', name: 'Create example graph', x: 100, y: 100 });
+  await workspace.addBox({
+    id: 'discard', name: 'Discard vertex attributes', x: 100, y: 200, after: 'ex'
+  });
+  const box = await workspace.openBoxEditor('discard')
+  await box.moveTo(500, 100);
+  const state = await workspace.openStateView('discard', 'graph');
+  await state.moveTo(500, 400);
+  const attrs = state.left.side.locator('entity[kind="vertex-attribute"]');
+  await expect(attrs).toHaveText(['age', 'gender', 'id', 'income', 'location', 'name'], { useInnerText: true });
```

Comment:
`useInnertText` comes from https://github.com/microsoft/playwright/issues/19072. I tried to make it the default by overriding `toHaveText` with `expect.extend`. Then I tried adding a new matcher. But it's not easy to extend existing matchers like this. (See https://github.com/facebook/jest/issues/6243 and https://github.com/facebook/jest/issues/10329.)

I went with just typing out `useInnerText` for now. We can look into cleaning this up later.